attributes/default.rb

```diff
@@ -42,18 +42,27 @@
 default['datadog']['agent_version'] = nil # nil to install latest
 # Agent flavor to install, acceptable values are "datadog-agent", "datadog-iot-agent"
 default['datadog']['agent_flavor'] = 'datadog-agent' # "datadog-agent" to install the datadog-agent package
+default['datadog']['fips_proxy_version'] = nil
+default['datadog']['agent_version'] = nil # nil to install latest
+# Datadog FIPS proxy package name:
+default['datadog']['fips_proxy_package_name'] = 'datadog-fips-proxy' # "datadog-fips-proxy" to install the datadog-fips-proxy package
 
 # Allow override with `upgrade` to get latest (Linux only)
 default['datadog']['agent_package_action'] = 'install'
+# Allow override with `upgrade` to get latest (Linux only)
+default['datadog']['fips_proxy_package_action'] = 'install'
 
 # Agent package options
 # retries and retry_delay for package download/install
 default['datadog']['agent_package_retries'] = nil
 default['datadog']['agent_package_retry_delay'] = nil
+default['datadog']['fips_proxy_package_retries'] = nil
+default['datadog']['fips_proxy_package_retry_delay'] = nil
```

Comment:
I would probably suggest using `agent_package_retries` and `agent_package_retry_delay` for both the Agent and the FIPS proxy. I know that the naming suggests that this would only apply to Agent, but I find it hard to believe that people would make these values different for Agent and FIPS proxy - most of the time, people will have to set both to exactly the same value and I don't think that's a great user experience. WDYT?